spec/civildate.html

```diff
@@ -1,164 +1,566 @@
-        1. Regulate the final y, m, and d values using Gregorian pro-leptic calendar calculations.
-        1. Let O be ? OrdinaryCreateFromConstructor(New Target, "%CivilDatePrototype%", y, m, d)
-        1. Set O.[[YearValue]] to y.
-        1. Set O.[[MonthValue]] to m.
-        1. Set O.[[DayValue]] to d.
-        1. Return O.
-      1. Else, TBD
-    </emu-alg>
+    <emu-clause>
+      <h1>get Temporal.Date.prototype.leapYear</h1>
+      <p>
+        Temporal.Date.prototype.leapYear is an accessor property whose set accessor function is undefined.
+        Its get accessor function performs the following steps:
+      </p>
+      <emu-alg>
+        1. Let _date_ be the *this* value.
+        1. If Type(_date_) is not Object or _date_ does not have an [[InitializedTemporalDate]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return ! IsLeapYear(_date_.[[Year]]) is *true*.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.Date.prototype.getYearMonth ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _date_ be the *this* value.
+        1. If Type(_date_) is not Object or _date_ does not have an [[InitializedTemporalDate]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return CreateYearMonth(_date_.[[Year]], _date_.[[Month]]).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.Date.prototype.getMonthDay ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _date_ be the *this* value.
+        1. If Type(_date_) is not Object or _date_ does not have an [[InitializedTemporalDate]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return CreateMonthDay(_date_.[[Month]], _date_.[[Day]]).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.Date.prototype.plus ( _duration_, _disambiguation_ )</h1>
+      <p>
+        The `plus` method takes one argument _duration_.
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _date_ be the *this* value.
+        1. If Type(_date_) is not Object or _date_ does not have an [[InitializedTemporalDate]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Let _duration_ be ? ToDurationLike(_duration_).
+        1. <mark>TODO: Throw if any time properties are present?</mark>
+        1. Let _disambiguation_ be ? ToString(_disambiguation_).
+        1. If _duration_.[[Year]] is not *undefined*, then
+          1. Let _y_ be _date_.[[Year]] + _duration_.[[Year]].
+        1. Else
+          1. Let _y_ be _date_.[[Year]].
+        1. If _duration_.[[Month]] is not *undefined*, then
+          1. Let _y_ be _date_.[[Month]] + _duration_.[[Month]].
+        1. Else
+          1. Let _y_ be _date_.[[Month]].
+        1. If _duration_.[[Day]] is not *undefined*, then
```

Comment:
* there's a Regulate step after adding months!
 * there is a Regulate step with "balance" at the end.
 * _m_ and _d_ not _y_

So it should be:

 1. Let _y_ be _date_.[[Year]] + _duration_.[[Years]]
 1. Let _m_ be _date_.[[Month]] + _duration_.[[Months]]
 1. Let _d_ be _date.[[Day]]
 1. Let _intermediate_ be ? RegulateDate(_y_, _m_, _d_, _disambiguation_)
 1. Let _d_ be _intermediate_.[[Day]] + _duration_.[[Days]]
 1. Let _result+ be ? RegulateDate(_intermediate_.[[Year]], _intermediate_.[[Month]], _d_, "balance")
 1. Return ? CreateDate(_result_.[[Year]], _result_.[[Month]], _result_.[[Day]], "balance")


And the analogous thing for `minus` (simply replace `+` with `-`)